### PR TITLE
Sunset Tube Network example

### LINKED ISSUE #49: Tube Network application has no example of TypeDB Reasoning
## Problem to Solve

The Tube Network application is not taking advantage of TypeDB Reasoning.

## Current Workaround

N/A

## Proposed Solution

Add a rule.
Suggestion from @jmsfltchr:
> we can use a rule to determine whether a train is eastbound or westbound

### PR DESCRIPTION
**Pull request by alexjpwalker**
## What is the goal of this PR?

We've sunsetted the Tube Network example due to its reliance on features of much older versions of TypeDB that are no longer supported.

## What are the changes implemented in this PR?

The Tube Network example used Grakn to perform OLAP and expressive graph analytics queries using Graql Compute queries. Given that we no longer support those queries, the example is no longer able to deliver significant value to the current TypeDB.

After discussion, we decided that it would require a lot of work in order to get the example to a working state, and even then, the queries it offered would not be as interesting as they could be - because in order for interesting questions to be answered in the route-planning space (Travelling Salesman problem, etc.), we need graph analytics queries, which TypeDB may support one day, but is not likely to support in the foreseeable future.

In conclusion, we are sunsetting the Tube Network example, noting that it can be restored from version control should TypeDB ever support the tools needed to make it an exciting application of TypeDB.

### CODE REVIEW

**Review by flyingsilverfin — APPROVED** [review]
Good!

**Comment by brettforbes** [comment]
hi @alexjpwalker and @dmitrii-ubskii ,

We would like to refurbish this example as a GIS mapping example, which we plan to do in [kepler.gl](https://[kepler.gl](https://kepler.gl/)/).

Can you give us access to the repo?

We will refurbish the loading code, but dump the analytics side, and focus on GIS visualisation. then we will publish the repo somewhere as open source, and we can add the analytics when you finally get that done

**Comment by jmsfltchr** [comment]
What access/why do you need access @brettforbes? I would have thought you can just clone the repo, checkout before the tube example deletion and copy the code over no?

**Comment by brettforbes** [comment]
aah yes @jmsfltchr , I got max barrass to copy it for me thanks, I actually found an old copy buried in my tests, and i already have it migrated.

It turns out the data set is brilliant because it contains both disjoint groups (station entity and properties) and joint groups (routes), combined with GIS, So its awesome for testing our open source mapping tool, thanks. We will end up reposting it as a demo example in our Type Explorer repo, hopefully with instructions to produce the tube map etc.
cheers
